tests/Tests/400__ods_dashboard/adminUITest.robot

```diff
@@ -0,0 +1,159 @@
+    Wait Until Page Contains    Launch application
+    Click Element               //*[@class="odh-card__footer__link"]
+    Wait Until Page Contains    Page Not Found   timeout=15
+    Page Should Contain         Page Not Found
+    [Teardown]  Teardown For Admin UI   ${TEST_USER.USERNAME}   ${TEST_USER.PASSWORD}  ${TEST_USER.AUTH_TYPE}
+
+*** Keywords ***
+Add OpenShift groups To Data Science Administrators
+    [Documentation]  Add OpenShift groups to Data Science administrators
+    [Arguments]     @{admin_groups}
+    Set Library Search Order  SeleniumLibrary
+    Click Button   (//button[@class="pf-c-button pf-c-select__toggle-button pf-m-plain"])[${1}]
+    FOR    ${admin_group}    IN    @{admin_groups}
+             ${present}=  Run Keyword And Return Status    Element Should Be Visible   //*[@class="pf-c-select__menu-item pf-m-selected" and contains(text(), "${admin_group}")]
+            IF  ${present} != True
+                Click Button    //*[@class="pf-c-select__menu-item"and contains(text(), "${admin_group}")]
+            END
+    END
+    Press Keys    None    ESC
+
+Add OpenShift Users groups To Data Science Administrators
+    [Documentation]  Add OpenShift Users groups to Data Science administrators
+    [Arguments]     @{user_groups}
+    Set Library Search Order  SeleniumLibrary
+    Click Button    (//button[@class="pf-c-button pf-c-select__toggle-button pf-m-plain"])[${2}]
+
+    FOR    ${user_group}    IN    @{user_groups}
+            ${present}=  Run Keyword And Return Status
+               ...  Element Should Be Visible   //*[@class="pf-c-select__menu-item pf-m-selected" and contains(text(), "${user_group}")]
+             IF  ${present} != True
+                Click Element    //*[@class="pf-c-select__menu-item"and contains(text(), "${user_group}")]
+             END
+    END
+    Press Keys    None    ESC
+
+Check User Management Option Is Available For The User
+    [Arguments]   ${username}  ${password}  ${auth_type}
+    Launch Dashboard  ocp_user_name=${username}  ocp_user_pw=${password}  ocp_user_auth_type=${auth_type}
+    ...               dashboard_url=${ODH_DASHBOARD_URL}  browser=${BROWSER.NAME}  browser_options=${BROWSER.OPTIONS}
+    Menu.Navigate To Page    Settings    User management
+    SeleniumLibrary.Wait Until Element Is Visible   //*[@class="pf-c-button pf-c-select__toggle-button pf-m-plain"]  timeout=20
+
+Remove OpenShift Users Groups To Data Science Administrators
+    [Documentation]     Remove User from OpenShift Users groups to Data Science administrators
+    [Arguments]   @{user_groups}
+    Set Library Search Order  SeleniumLibrary
+    FOR    ${user_group}    IN    @{user_groups}
+            Click Element     (//*[@class="pf-c-form__group-control"])[${2}]//*[@class="pf-c-chip__text" and contains(text(),"${user_groups}")]//following-sibling::button[${1}]
+    END
+    Press Keys    None    ESC
+
+Remove OpenShift Groups To Data Science Administrators
```

Comment:
are you removing admin groups or user (i.e. allowed) groups?

Reply:
`Remove OpenShift Group From Data Science Administrator Groups`

Reply:
changed